webclient/uzivatel/templates/admin/admin_user_change.html

```diff
@@ -1,6 +1,14 @@
 {% extends "admin/change_form.html" %}
 {% load i18n admin_modify %}
 
+{% block object-tools-items %}
+  <li class="objectaction-item" data-tool-name="metadata">
+    <a href="/admin/uzivatel/user/{{ object_id }}/actions/metadata/?_changelist_filters=o%3D-1"
```

Comment:
Má tam fakt být i to `?_changelist_filters=o%3D-1`?

Reply:
Není to potřeba